timelock-server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM frolvlad/alpine-oraclejdk8:slim
+MAINTAINER AtlasDB Team
+
+RUN apk update && apk add libstdc++ bash wget
+
+# Prefer ADD to COPY because it does the tar handling
+ADD build/docker/timelock-server-snapshot.tgz /timelock-server
+
+# Remove possible version prefix
+RUN for file in timelock-server/*; do mv "$file" "timelock-server/timelock-server-snapshot"; done
```

Comment:
Indeed we can. I copied this from how we do it in the Atlas ETE Server, but we can do better.